ui/nuxeo-document-comments/nuxeo-document-comment.js

```diff
@@ -0,0 +1,501 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import '@polymer/iron-flex-layout/iron-flex-layout.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/paper-button/paper-button.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-item/paper-icon-item.js';
+import '@polymer/paper-input/paper-textarea.js';
+import '@polymer/paper-listbox/paper-listbox.js';
+import '@polymer/paper-menu-button/paper-menu-button.js';
+import '@nuxeo/nuxeo-elements/nuxeo-connection.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '@nuxeo/nuxeo-elements/nuxeo-resource.js';
+import '../widgets/nuxeo-dialog.js';
+import '../widgets/nuxeo-tooltip.js';
+// eslint-disable-next-line import/no-cycle
+import './nuxeo-document-comment-thread.js';
+import './nuxeo-document-comments-styles.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import { FormatBehavior } from '../nuxeo-format-behavior.js';
+import CommentsMixin from './nuxeo-document-comments-mixin.js';
+
+/**
+ * Element to represent a comment.
+ * It shows comment's data like: author, date, text and available options (reply, edit, delete).
+ *
+ * Example:
+ *
+ *     <nuxeo-document-comment comment="[[comment]]"></nuxeo-document-comment>
+ *
+ * @appliesMixin Nuxeo.CommentsMixin
```

Comment:
After the comments above:
```suggestion
 * @appliesMixin CommentsMixin
```